x-pack/plugins/maps/public/classes/sources/ems_tms_source/ems_tms_source.tsx

```diff
@@ -66,7 +66,7 @@ export class EMSTMSSource extends AbstractSource implements ITMSSource {
   }
 
   async getImmutableProperties() {
-    const displayName = await this.getDisplayName();
+    const tileServiceName = await this._getTileServiceName();
     const autoSelectMsg = i18n.translate('xpack.maps.source.emsTile.isAutoSelectLabel', {
       defaultMessage: 'autoselect based on Kibana theme',
```

Comment:
Maybe for another PR, but can we remove or use something other than Kibana here?